draft-ietf-rtcweb-jsep.xml

```diff
@@ -1140,7 +1140,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="addTrack" anchor="sec.addTrack">
-                  <t>[TODO]</t>
+                  <t>
+                    The addTrack method adds a MediaStreamTrack to the PeerConnection.
+                    addTrack attempts to minimize the number of transceivers as follows:
+                    The track will be attached to the first compatible transceiver
+                    (of the same media type) which has never had a direction of
+                    "sendonly" or "sendrecv". If no such transceiver exists,
```

Comment:
Justin, here is the text I am relying on:

"If an RTCRtpSender exists in connection's set of senders that has never been used to send (the corresponding media description has always had a direction of recvonly or inactive), then set that sender's track to track and return the sender.

Doing so will cause future calls to createOffer and createAnswer to mark the corresponding media description as sendrecv or sendonly, as defined in [JSEP]."